spec/controllers/dashboard_controller_spec.rb

```diff
@@ -119,4 +119,30 @@
       end
     end
   end
+
+  describe 'Check Primary Account' do
+    let(:admin) { create :user, :admin }
```

Comment:
And since there's only one test per context, you can just write

```ruby
when_current_user_is :student
```

and

```ruby
when_current_user_is :admin
```